packages/stream-tile-handler/src/tile-document-handler.ts

```diff
@@ -210,8 +217,41 @@ export class TileDocumentHandler implements StreamHandler<TileDocument> {
   async _verifySignature(
```

Comment:
Don't forget to update the doccomments when changing a function signature!  You should be able to get your IDE to give you a warning whenever a function's signature and doccomments don't match

Reply:
will do! thanks :D